pytorch/containers/custom_container/README.md

```diff
@@ -0,0 +1,165 @@
+
+[Cloud ML Engine](https://cloud.google.com/ml-engine/) is a managed service that enables you to easily build machine learning models that work on any type of data, of any size.
+
+[Cloud Container Registry](https://cloud.google.com/container-registry/) is a single place for your team to manage Docker images, perform vulnerability analysis, and decide who can access what with fine-grained access control.
+
+[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is a unified object storage for developers and enterprises, from live data serving to data analytics/ML to data archiving.
+
+[Cloud SDK](https://cloud.google.com/sdk/) is a command line tool which allows you to interact with Google Cloud products. In order to run this notebook, make sure that Cloud SDK is [installed](https://cloud.google.com/sdk/downloads) in the same environment as your Jupyter kernel.
+
+[docker](https://www.docker.com/) is a containerization technology that allows developers to package their applications and dependencies easily so that they can be run anywhere.
+
+# Part 0: Setup
+* [Create a project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
+* [Create a Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/quickstart-console)
+* [Enable Cloud Machine Learning Engine, Container Registry, and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component,containerregistry.googleapis.com)
+* [Install Cloud SDK](https://cloud.google.com/sdk/downloads)
+* [Install docker](https://docs.docker.com/install/)
+* [Configure docker for Cloud Container Registry](https://cloud.google.com/container-registry/docs/pushing-and-pulling)
+* [Install PyTorch](https://pytorch.org/get-started/locally/) [Optional: used if running locally]
+* [Install pandas](https://pandas.pydata.org/pandas-docs/stable/install.html) [Optional: used if running locally]
+
+These variables will be needed for the following steps.
+
+**Replace**
+* `PROJECT_ID <YOUR_PROJECT_ID>` - with your project's id. Use the PROJECT_ID that matches your Google Cloud Platform project.
+* `BUCKET_ID <YOUR_BUCKET_ID>` - with the bucket id you created above.
+* `IMAGE_REPO_NAME <sonar_pytorch_container>` - where the image will be stored on Cloud Container Registry
+* `IMAGE_TAG <sonar_pytorch>` - a easily identifiable tag for your docker image
+* `IMAGE_URI <gcr.io/$PROJECT_ID/$IMAGE_REPO_NAME:$IMAGE_TAG>` - the complete URI location for Cloud Container Registry
+* `REGION <REGION>` - select a region from [here](https://cloud.google.com/ml-engine/docs/regions) or use the default '`us-central1`'. The region is where the model will be deployed.
+* `JOB_NAME <custom_container_job_...>` - the name of your job running on Cloud ML Engine.
+* `MODEL_NAME <sonar_model>` - the name of your saved model file
+
+```
+# Replace YOUR_PROJECT_ID
```

Comment:
Having the `PROJECT_ID` and `BUCKET_ID` in a separate code block might make it more obvious that those are the variables that *need* to be changed while the others could be left as is.

Reply:
Will do